pkg/operator/operator/validations.go

```diff
@@ -534,11 +546,18 @@ func validateTensorFlowPredictor(predictor *userconfig.Predictor) error {
 			return errors.Wrap(ErrorS3FileNotFound(model), userconfig.ModelKey)
 		}
 	} else {
-		path, err := getTFServingExportFromS3Path(model, awsClient)
+		neuronExport := false
+		if api.Compute.ASIC > 0 {
+			neuronExport = true
+		}
+		path, err := getTFServingExportFromS3Path(model, neuronExport, awsClient)
 		if err != nil {
 			return errors.Wrap(err, userconfig.ModelKey)
 		} else if path == "" {
-			return errors.Wrap(ErrorInvalidTensorFlowDir(model), userconfig.ModelKey)
+			if !neuronExport {
```

Comment:
nit: I think it might be a little more readable if we switch the order, e.g. `if neuronExport {...`